README.md

```diff
@@ -44,6 +44,35 @@ done.
 Wrote build script to .spago2nix/build
 ```
 
+When using in your own Nix derivation, the best practice is calling generated scripts from `spago-packages.nix`:
+
+```nix
+{ pkgs, stdenv }:
+
+let 
+  spagoPkgs = import ./spago-packages.nix { inherit pkgs; };
+in
+pkgs.stdenv.mkDerivation rec {
+  # < ... >
+  buildPhase = 
+  '' 
+    ${spagoPkgs.installSpagoStyle} # == spago2nix install
+    ${spagoPkgs.buildSpagoStyle}   # == spago2nix build
```

Comment:
Should add a note to the build that you need to provide input paths

https://github.com/justinwoo/spago2nix/blob/d97c9a4585ef563bc8c16f2904d0a8035293a508/spago-packages.nix#L824

`$@` is provided here to feed path inputs into the script